librz/core/cprint.c

```diff
@@ -539,6 +539,32 @@ static void core_handle_call(RzCore *core, char *line, char **str) {
 	}
 }
 
+/**
+ *  \brief Get the console output of disassembling \p byte_len bytes at \p addr
+ */
+static char *cons_dis_n_bytes(RzCore *core, ut64 addr, ut32 byte_len) {
+	ut8 *block = malloc(byte_len + 1);
+	if (!block) {
+		RZ_LOG_ERROR("Cannot allocate buffer\n");
+		return NULL;
+	}
+
+	rz_io_read_at(core->io, addr, block, byte_len);
```

Comment:
Eventually we will get rid of the blocks: https://github.com/rizinorg/rizin/issues/2695
But for now is fine.

Reply:
Thanks for reminding me. I will pay attention to that.